packages/gitbook/src/components/Header/HeaderLogo.tsx

```diff
@@ -21,6 +21,7 @@ export async function HeaderLogo(props: HeaderLogoProps) {
     return (
         <Link
             href={linker.toAbsoluteURL(linker.toPathInSite(''))}
+            target="_self"
```

Comment:
I don't understand why there is a case where it was not opening in the same tab.

Reply:
It's live on https://docs.ionos.com/cloud/